ChunkArray/chunkarray.js

```diff
@@ -0,0 +1,31 @@
+var chunk = function (arr, size) {
+  let start = 0;
+  let ans = [];
+
+  if(size == 0){
```

Comment:
```suggestion
  if(size === 0){
```
always use === instead of ==. this will insure type check as well.